src/docs/truffle/getting-started/package-management-via-ethpm.md

```diff
@@ -68,68 +94,84 @@ module.exports = function(deployer) {
 
 Note that in the migration above, we consume the `ens` package and deploy the ENS contract conditionally based on whether or not ENS already has an address set. This is a fancy trick provided to you by the [deployer](/docs/getting_started/migrations#deployer-deploy-contract-args-options-) that makes it much easier to write migrations dependent on the the existence of network artifacts. In this case, if we were running our migrations on the Ropsten network, this migration **wouldn't** deploy the `ENS` contract because (at the time of this writing) Ropsten is where the canonical `ENS` contract exists -- we wouldn't want to deploy our own. But if we were running our migrations against a different network, or a test network perhaps, then we'd want to deploy the `ENS` contract so that we have a dependency contract to work with.
 
+### 5. Interacting with deployments.
+Ethpm packages often contain on-chain deployments, so you can immediately interact with verified contract instances. After you've installed a package, simply fire up your `truffle console` and run the `networks` command to see the available instances.
+
 ## Publishing your own package
 
-Publishing your own package is as straightforward as installing, but like NPM, requires a bit more configuration.
+Publishing your own package is as straightforward as installing, but like NPM, requires a bit more configuration. However, currently Truffle only supports publishing solidity contracts.
+
+### Setup
 
-### Ropsten, Ropsten, Ropsten
+By default, your package will be published to the un-authorized, default registry. This registry must not be used in production, and was only created to simplify experimenting with ethpm. We strongly recommend that you deploy your own, authorized ethpm registry to host your packages.
 
-The Ethereum Package Registry currently exists on the Ropsten test network. To publish to the registry, we need to set up our own Ropsten configuration because we'll be making transactions that need to be signed.
+To publish to any registry, we need to set up our configuration to make signed transactions. If you are publishing to an authorized registry, the signing address must also be authorized to cut a release on the connected registry.
```

Comment:
What exactly does it mean "to be authorized to cut a release on the connected registry"?

Reply:
All ethpm registries are permissioned with a general "owner" model (except for the default registry shipping with truffle that is non-permissioned, and therefore not safe to use). Only the "owner" address is allowed to "cut a release" on a registry. Assuming an honest owner, you can trust all of the releases on their registry to be safe to use. This prevents a malicious party from publishing unsafe packages to a registry. It's a somewhat important aspect of ethpm, because it's very unsafe to interact with packages from "untrusted" registries. Lmk if that doesn't really come across in the docs, and I'll expand on it further.